packages/@aws-cdk/aws-eks/lib/helm-chart.ts

```diff
@@ -53,6 +53,12 @@ export interface HelmChartOptions {
    * @default Duration.minutes(5)
    */
   readonly timeout?: Duration;
+
+  /**
+   * create namespace if not exist
+   * @default False
+   */
+  readonly createNamespace?: boolean;
```

Comment:
Feels like it makes more sense for this to be enabled by default, no? What are the implications?